iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -66,6 +74,7 @@
   private boolean deleteIcebergTable;
   private FileIO deleteIo;
   private TableMetadata deleteMetadata;
+  private boolean generateMetadata;
```

Comment:
How about a different naming that reflects that it's related to "migratability"? e.g. `boolean canMigrateHiveTable`